tests/fixers/test_index_together.py

```diff
@@ -0,0 +1,597 @@
+                indexes = []
+        """,
+        """\
+        from django.db import models
+
+        class Duck(models.Model):
+            class Meta:
+                indexes = [models.Index(fields=("bill", "tail"))]
+        """,
+    )
+
+
+def test_tuple_single_indexes_present():
+    check_transformed(
+        """\
+        from django.db import models
+
+        class Duck(models.Model):
+            class Meta:
+                index_together = ("bill", "tail")
+                indexes = []
+        """,
+        """\
+        from django.db import models
+
+        class Duck(models.Model):
+            class Meta:
+                indexes = [models.Index(fields=("bill", "tail"))]
+        """,
+    )
+
+
+def test_mixed_indexes_present():
+    check_transformed(
+        """\
+        from django.db import models
+
+        class Duck(models.Model):
+            class Meta:
+                index_together = [
+                    ("bill", "tail"),
+                    ("nape", "mantle"),
+                ]
+                indexes = []
+        """,
+        """\
+        from django.db import models
+
+        class Duck(models.Model):
+            class Meta:
+                indexes = [models.Index(fields=("bill", "tail")), models.Index(fields=("nape", "mantle"))]
+        """,
+    )
+
+
+def test_indexes_nonempty_no_trailing_comma():
+    check_transformed(
+        """\
+        from django.db import models
+
+        class Duck(models.Model):
+            class Meta:
+                index_together = [("bill", "tail")]
+                indexes = [models.Index("bill")]
+        """,
+        """\
+        from django.db import models
+
+        class Duck(models.Model):
+            class Meta:
+                indexes = [models.Index("bill"), models.Index(fields=("bill", "tail"))]
+        """,
+    )
+
+
+def test_indexes_nonempty_trailing_comma():
+    check_transformed(
+        """\
+        from django.db import models
+
+        class Duck(models.Model):
+            class Meta:
+                index_together = [("bill", "tail")]
+                indexes = [models.Index("bill"),]
+        """,
+        """\
+        from django.db import models
+
+        class Duck(models.Model):
+            class Meta:
+                indexes = [models.Index("bill"), models.Index(fields=("bill", "tail"))]
+        """,
+    )
+
+
+def test_indexes_nonempty_multiline_dedented():
+    check_transformed(
+        """\
+        from django.db import models
+
+        class Duck(models.Model):
+            class Meta:
+                index_together = [("bill", "tail")]
+                indexes = [
+                    models.Index("bill"),
+                ]
+        """,
+        """\
+        from django.db import models
+
+        class Duck(models.Model):
+            class Meta:
+                indexes = [
+                    models.Index("bill"),
+                models.Index(fields=("bill", "tail"))]
```

Comment:
Funky indentation here and below 😛 

But any reasonable project that uses a formatter, may it by black or ruff, should be fine :)

Reply:
Yeah, the README explicitly says:

> django-upgrade focuses on upgrading your code and not on making it look nice. Run django-upgrade before formatters like Black.

It’s hard enough to ensure generated code is valid, nice formatting is an extra level and just duplicative of black etc. anyway.